.eslintrc.js

```diff
@@ -95,5 +98,34 @@ module.exports = {
 		"jsx-a11y/anchor-has-content": 0,
 		"react/no-string-refs": 0,
 		"jsx-a11y/anchor-is-valid": 0,
-	}
+	},
+	overrides: [
+		{
+			files: [ "extensions/**/*", "webpack.config.extensions.js" ],
+			rules: {
+				// Deprecated rule, fails in some valid cases with custom input components
+				"jsx-a11y/label-has-for": 0,
+				"lodash/import-scope": 0,
+				"react/forbid-elements": [
+					"error",
+					{
+						forbid: [
+							[ "circle", "Circle" ],
+							[ "g", "G" ],
+							[ "path", "Path" ],
+							[ "polygon", "Polygon" ],
+							[ "rect", "Rect" ],
+							[ "svg", "SVG" ],
+						].map( ( [ element, componentName ] ) => ( {
+							element,
+							message: `use <${ componentName }> from @wordpress/components`,
+						} ) ),
+					},
+				],
+				"react/jsx-no-bind": 0,
+				"react/react-in-jsx-scope": 0,
+				"wpcalypso/jsx-classname-namespace": 0,
+			},
+		},
+	]
```

Comment:
Hmm, so we didn't include `jetpack-blocks`' [`.eslintrc.js`](https://github.com/Automattic/wp-calypso/blob/24daf03893351349cc59100310f85173d867100a/packages/jetpack-blocks/.eslintrc.js) in the move? :slightly_frowning_face:

Reply:
Rules from that config are fairly small but it seems like because this PR was originally against another branch than `master`, CI linter didn't run at all so we didn't catch other linter issues here.

Reply:
I brought changes over from jetpack-blocks' .eslintrc.js in https://github.com/Automattic/jetpack/pull/11640/commits/da2c4515aa6dc708a1170ad46b542ea77f344cb1

Reply:
No, the lint errors didn't pop up and we didn't pull it in. Ideally we share published prettier and eslint from Calypso, but we're not quite there. I'd like this entire eslintconfig to be something like `extends: [ 'wp-calypso/react', 'wp-calypso/gutenberg' ]`.